src/test/kotlin/no/nav/syfo/testhelper/ExternalMockEnvironment.kt

```diff
@@ -20,4 +20,22 @@ class ExternalMockEnvironment {
     val environment: Environment = testEnvironment()
 
     val wellKnownInternalAzureAD = wellKnownInternalAzureAD()
+
+    val redisServer = testRedisServer(redisConfig = environment.redis)
```

Comment:
Hva gjør vi annerledes nå som gjør at vi trenger denne? Trodde vi mocket alle redis-instansene i testene?

Reply:
Usikker, var noen tester som begynte å feile, så dette var den løsningen jeg fant.

Reply:
Da får jeg ikke kjørt tester lokalt lenger 😢 RedisServer i testene henger seg alltid opp for meg, så jeg må starte maskinen på nytt for å få kjørt tester 🤷‍♂️

Reply:
OK, forsøker å få det bort igjen i en egen PR.